### DISCUSSION TITLE
Setup Prow to enable prow command integrations

### DISCUSSION THREAD

**Discussion by usrbinkat** [emporous/community#8] [Ideas]
Currently, the ability to request reviews and assign issues is limited.

I can help with this integration but I could use some pointers and examples to emulate.

**Comment by jpower432** [comment]
Hey @usrbinkat. Thanks for starting this discussion. A great way to start talking about this is to list the capabilities we would like to see on the organization/repo.

Here are some automation that I would love to see:
1. Automated PR merging
2. Command integration for labeling issues and PRs
3. Command integration for running a particular suite of tests
